newm-server/src/main/kotlin/io/newm/server/features/song/repo/SongRepositoryImpl.kt

```diff
@@ -377,7 +377,7 @@ internal class SongRepositoryImpl(
         logger.info { "sent: $messageToSend" }
 
         when (mintingStatus) {
-            MintingStatus.MintingPaymentReceived -> {
+            MintingStatus.MintingPaymentRequested -> {
```

Comment:
Is this something we want to do? What if they spam their collaborators, but then don't pay?

Reply:
I see the potential issue. Let's discuss alternatives via Slack.